cfgov/unprocessed/css/apps/owning-a-home/form-explainer.less

```diff
@@ -0,0 +1,447 @@
+/* ==========================================================================
+   "Form explainer" custom styles
+   ========================================================================== */
+
+.form-explainer {
+    .u-webfont-regular();
+
+    .u-right-on-desktop {
+        .respond-to-min( 801px, {
+            text-align: right;
+        } );
+    }
+
+    .page-intro {
+        padding-bottom: 36px;
+    }
+
+    .illustration {
+        padding-top: 1em;
+        width: 200px;
+        margin: 0 auto;
+        display: block;
+        .respond-to-max( 719px, {
+            display: none;
+        } );
+    }
+
+    .rateq {
+        margin-top: 0px;
+    }
+
+    .o-expandable_header,
+    .o-expandable_content {
+        background: transparent;
+    }
+
+    .image-map {
+        display: block;
+        box-sizing: border-box;
+        margin-bottom: unit( 30px / @base-font-size-px, em );
+        .u-clearfix();
+    }
+
+    .image-map_wrapper {
+        float: left;
+        position: relative;
+    }
+
+    .image-map_image {
+        box-sizing: border-box;
+        display: block;
+        border: 1px solid @black;
+    }
+
+    .image-map_overlay {
+        position: absolute;
+        border: 2px solid transparent;
+        background: transparent;
+        transition: border 100ms ease-out;
+        box-sizing: border-box;
+    }
+
+    .image-map_overlay__checklist {
+        & {
+            background: rgba( 255, 147, 27, 0.2 );
+        }
+        &:hover,
+        &.hover-has-attention {
+            border-color: @gold-80;
+        }
+        &:focus,
+        &.has-attention {
+            border-color: @gold-80;
+            background: rgba( 255, 147, 27, 0.5 );
+        }
+        &:focus {
+            outline: 1px dotted @gold;
+            outline-offset: 1px;
+        }
+    }
+
+    .lt-ie9 .image-map_overlay__checklist {
+        background: none;
+        -ms-filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#33FF931B,endColorstr=#33FF931B)";
+        filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#33FF931B,endColorstr=#33FF931B)";
```

Comment:
Any chance we can get autoprefixer to handle this and the prefixed version above?

Reply:
Code was removed.